docs/content/en/docs/user-guide/examples/k8s-app-canary-with-istio.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Canary deployment for Kubernetes app with Istio"
+linkTitle: "Canary k8s app with Istio"
+weight: 1
+description: >
+  How to enable canary deployment for Kubernetes application with Istio.
+---
+
+"Canary release is a technique to reduce the risk of introducing a new software version in production by slowly rolling out the change to a small subset of users before rolling it out to the entire infrastructure and making it available to everybody."
```

Comment:
nit, using quotes for a better look 👀 
```
> Canary release is a technique to reduce the risk of introducing a new software version in production by slowly rolling out the change to a small subset of users before rolling it out to the entire infrastructure and making it available to everybody.
```

Reply:
Sure. Which style do you prefer?

A.
![Screen Shot 2021-02-02 at 11 39 55](https://user-images.githubusercontent.com/1751755/106544914-b21b0580-654b-11eb-9ec2-06b6cb321ac0.png)

B.
![Screen Shot 2021-02-02 at 11 41 00](https://user-images.githubusercontent.com/1751755/106544919-b5ae8c80-654b-11eb-9f5c-40364f14be24.png)

Reply:
I don't know how to be shown on the hugo server exactly but sounds good.

Reply:
Again, browser refresh stuff.

Can't be nested quotes on Hugo right?

Reply:
@nakabonne refresh your chrome tab and you will see @nghialv suggestion example 😂 
How about this https://stackoverflow.com/a/2002150/5938316

```
> Quote here.
>
> -- <cite>Benjamin Franklin</cite>
```

Reply:
Nice. How about this? Looks good.
![Screen Shot 2021-02-02 at 11 57 36](https://user-images.githubusercontent.com/1751755/106546097-ea234800-654d-11eb-84f0-184e500010e7.png)

Reply:
Cool 👍 But I think to add `from` as `from martinfowler.com/canaryrelease` maybe better 😂

Reply:
![Screen Shot 2021-02-02 at 12 02 06](https://user-images.githubusercontent.com/1751755/106546431-8baa9980-654e-11eb-8707-56452d0319b8.png)

Reply:
Maybe this is better. Don't need "from" because of the same line.

Reply:
Or in the new line with "From". lol

Reply:
I will go for this 💯 
https://github.com/pipe-cd/pipe/pull/1528#discussion_r568290360